Cargo.toml

```diff
@@ -1,6 +1,15 @@
 [workspace]
 resolver = "2"
 members = [
+  "crates/*",
+  "editors/zed",
+  "extensions/*",
+  "rust/*",
+  "tombi-wasm",
+  "toml-test",
+  "xtask",
+]
+default-members = [
```

Comment:
Instead of using default-members,
I would like to build with `default-features = false` in `tombi-wasm`.

Reply:
Since it's not possible to set `default-features` on dependencies in the workspace root, are you suggesting something like explicitly specifying a `wasm` feature at build-time by adding this to `tombi-wasm/Cargo.toml`?

```toml
[features]
default = []
wasm = ["serde_tombi/wasm", "tombi-schema-store/wasm", "tombi-wasm-compat/wasm"]
```